R/integer64.R

```diff
@@ -1067,6 +1092,32 @@ seq.integer64 <- function(from=NULL, to=NULL, by=NULL, length.out=NULL, along.wi
   return(ret)
 }
 
+
+# helper for determining the target class for Ops methods
+target_class_for_Ops = function(e1, e2) {
```

Comment:
Is it possible for this to be a `chooseOpsMethod()` method?

https://stat.ethz.ch/R-manual/R-devel/library/base/html/chooseOpsMethod.html